src/main/main.ts

```diff
@@ -1,21 +1,22 @@
-import { initSentry } from '../sentry';
-initSentry();
```

Comment:
The sentry should be left as it is, the sooner the better

Reply:
Oh Cool! Wasn't aware of it :)